src/core/observables/CylindricalLBProfileObservable.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef OBSERVABLES_CYLINDRICALLBPROFILEOBSERVABLE_HPP
+#define OBSERVABLES_CYLINDRICALLBPROFILEOBSERVABLE_HPP
+
+#include <cmath>
+
+#include "CylindricalProfileObservable.hpp"
+#include "Vector.hpp"
+#include "integrate.hpp"
+#include "particle_data.hpp"
+#include "utils/coordinate_transformation.hpp"
+
+namespace Observables {
+
+class CylindricalLBProfileObservable : public CylindricalProfileObservable {
+public:
+  double sampling_delta_x=1.0, sampling_delta_y=1.0, sampling_delta_z=1.0;
```

Comment:
Most of this should be private.

Reply:
You mean I should implement getter and setter?